simulation/modelica/initialization/discreteTest06.mos

```diff
@@ -17,16 +17,18 @@ package initializationTests
   initial equation
     pre(y) = 0;
 
-  equation 
+  equation
     der(x) = y;
 
+    // Disallowed in MLS 3.2r1
```

Comment:
But the comment suggests that the test is specifically for the case of using or instead of the better array syntax. So I'll keep it as it was. At least we then check for old (allowed in 3.1) code.